src/style.css

```diff
@@ -1,3 +1,74 @@
+:root {
+  --active-color: #00acdc;
+}
+
 body {
   margin: 0;
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: 'Roboto', serif;
```

Comment:
The font-family declaration uses 'Roboto' with quotes. Ensure that this matches the HTML file where 'Roboto' is imported without quotes. Consistency in font-family usage is important to avoid potential rendering issues.